modules/exploits/windows/ftp/quickshare_traversal_write.rb

```diff
@@ -147,11 +146,16 @@ def exploit
 
     begin
       t = framework.threads.spawn("reqs", false) {
-        # Upload our malicious executable
-        u = upload(exe_name)
-
-        # Upload the mof file
-        upload(mof_name) if u
+        begin
+          # Upload our malicious executable
+          u = upload(exe_name)
+
+          # Upload the mof file
+          upload(mof_name) if u
+        rescue ::Exception => e
+          print_error e.message
```

Comment:
Would this be useful information to also `elog(e)` for error reports?

Reply:
This in the module itself, I believe the `print_error` should be sufficient.